app/backend/src/couchers/servicers/api.py

```diff
@@ -302,9 +302,7 @@ def UpdateProfile(self, request, context):
                     )
 
             if request.HasField("regions_visited"):
-                for region in user.regions_visited:
-                    session.delete(region)
-                session.flush()
+                session.execute(delete(RegionVisited).where(RegionVisited.user_id == context.user_id))
```

Comment:
Does this sync session?

Reply:
I'm not sure I understand?

Reply:
Don't worry, I didn't think that comment through.

Same question as with flush: does it work this way? :P

Reply:
Certainly seems to. All tests pass. And I added a check to make sure that the regions delete and then get re-added without violating unique constraints